server/schedule/operator/builder.go

```diff
@@ -514,7 +528,9 @@ func (b *Builder) setTargetLeaderIfNotExist() {
 	}
 
 	leaderPreferFuncs := []func(uint64) int{
+		b.preferLeaderRoleAsLeader,
 		b.preferUpStoreAsLeader,
+		b.preferCurrentLeader,
```

Comment:
There is duplication with the code function in `prepareBuild`, which seems to cause it to become the highest priority. We need to discuss this.